hello/lib/hello_web/plugs/locale.ex

```diff
@@ -0,0 +1,15 @@
+defmodule HelloWeb.Plugs.Locale do
```

Comment:
@Vretamoza what does a plug do?

Reply:
A plug is a specification that helps to build endpoints. It receives a conn, modifies it and return it.